lib/icalendar_helper.rb

```diff
@@ -0,0 +1,40 @@
+require 'icalendar' # https://github.com/icalendar/icalendar
+
+# Includes all abstractions over iCalender representations
+module IcalendarHelper
+  # Converts a sticky activity to an iCalendar event
+  def self.activity_to_event(activity, locale)
+    event = Icalendar::Event.new
+    event.uid = activity.id.to_s
+    event.dtstart = activity.start_date
+    event.dtend = activity.end_date
+    event.summary = activity.name
+    event.description = activity.description_localised(locale)
+    event.location = activity.location
+    return event
+  end
+
+  # Combines zero or more Icalendar events into an iCalendar abstract object
+  def self.create_calendar(events, locale)
+    calendar = Icalendar::Calendar.new
+    calendar.x_wr_calname =
+      case locale
+      when :nl
+        "Sticky Activiteiten"
+      else
+        "Sticky Activities"
+      end
```

Comment:
This seems overly verbose, i reckon it would be better to put these strings in the i18n file and reference that